_episodes/licenses.md

```diff
@@ -1,24 +1,149 @@
+- <https://www.openaire.eu/research-data-how-to-license/>
+
+> ## Excercise
+>- When was the last time you published some data?
+>- What licence did you distribute it under?
+>- Where did you share the data?
+{: .discussion}
+
+Licences in Geoscience
+----------------------
+
+In general, data sources in geoscience such as CMIP5 and CMIP6 are available under FAIR licences. That is, you can freely use the data, but there are still some requirements. For example, if you base your research on CMIP6 data, you must give proper credit to the project if you deposit your research data somewhere. This is specified in the Terms of Use.
+
+You can find the Terms of Use for some widely used projects here:
+
+- CMIP6: <https://pcmdi.llnl.gov/CMIP6/TermsOfUse/TermsOfUse6-1.html>
+- CMIP5: <https://pcmdi.llnl.gov/mips/cmip5/terms-of-use.html>
+- CORDEX: <http://is-enes-data.github.io/cordex_terms_of_use.pdf>
+
+>## Exercise
+>- Where can you find the licence for CMIP6 data? What about CMIP5?
+>- How do the terms of use for CMIP6 data differ from the Creative Commons licences?
+>- Can you reshare the data?
+>- Should you attribute the authors of the data? If so, how?
+>- Can derived data be used commecially?
+>- **FIXME**: check what the answer should be
+{: .discussion}
+
+Further reading:
+- https://gmd.copernicus.org/articles/11/3659/2018/ (Section 4)
+
+
+
+Data ownership
+--------------
+
+Possible owners of the data:
+- You
+- Institute you work for
+- Funding agency
+
+
+You may be restricted by your institutions’ policies on licenses
+
+Data ownership is defined in the data management plan
+
+
+> ## Excercise
+>Take a few minutes to find out whether your institute has a license policy.
+>
+>- What does it say?
+>- Who owns *your* data?
+{: .discussion}
+
+> ## Excercise
+>- Imagine you have downloaded some data from source X and applied operation Y on it.
+>- Which data licence should you put on it when you distribute the data?
+>- Which licence can you not put on it when you distribute the data?
+{: .discussion}
+
+
+What does the EU require?
+-------------------------
+
+The European Commission has fully embraced the FAIR principles. Horizon2020 already mandates open access to all scientific publications, and is now running a pilot from 2017 for research data to be open by default with possibilities to opt out.
+
+How the data will be licenced is part of the data management plan. Specificically:
+
+- How will the data be licensed to permit the widest re-use possible?
+- When will the data be made available for re-use (i.e. are the data under embargo)?
+- Are the data usable by third parties (i.e. are there restrictions)?
+
+In all cases, the Commission encourages researchers to provide access to their data in the broadest sense. They encourage authors to retain their copyright, but grant adequate licences to publishers. They recommend the Creative Common licences as a useful legal tool for providing open access.
+
+Further reading:
```

Comment:
Perhaps we can create an exercise out of this? Let them read the document and find the info themselves? In my experience it is easier to memorize "where to find info" than to memorize the actual info